server/api/user/user.controller.ts

```diff
@@ -189,6 +190,37 @@ class UserController {
     res.json(removedUser);
   }
 
+  async removeUser(req: ApiRequest<OrgEdipiParams>, res: Response) {
+    if (!req.appOrg) {
```

Comment:
These checks still exist throughout the API, but they are no longer necessary, you can remove them from controller functions whenever you see them.  This check is being handled in the `requireOrgAccess` middleware.

Reply:
Awesome. Removing...